### DISCUSSION TITLE
Row Update

### DISCUSSION THREAD

**Discussion by jonnydayy** [Budibase/budibase#6014] [Help]
Is it possible to load row data and 'lock' the data from others from using? Ie FOR UPDATE locking at start of data load and releasing when updated?

**Comment by melohagan** [comment]
Hi @jonnydayy 

Are you using the internal Budibase DB? 

Here's a StackOverflow post on how revisions work in the context of DB locking for CouchDB: https://stackoverflow.com/questions/299723/can-i-do-transactions-and-locks-in-couchdb
